packages/core/src/crypto/jose/jwa/alg.ts

```diff
@@ -0,0 +1,38 @@
+export enum JwaSignatureAlgorithm {
```

Comment:
Do we want to have all the IANA / IETF defined algorithms here or just that we support?

Reply:
I would say we add them all, as that makes it easier to include them in the future. If no JWK / JWE / JWS refers to it, it wouldn't be able to be used.

but if you want, I can remove the unused ones

Reply:
No, it's okay. Main point was to avoid confusion about what we support, but it should be fine.